docs/develop/topics/administration_panel.md

```diff
@@ -0,0 +1,432 @@
+To find more about the different blocks that can be overridden in each of the views you can check the code of the views [here](https://github.com/inveniosoftware/invenio-administration/tree/main/invenio_administration/templates/semantic-ui/invenio_administration).
+
+### Customisation: React components
+
+In addition to jinja, in each view there is the possibility to inject React components. In order to do this, the view must be extended with a custom jinja template as described above. In this template, the javascript code that will render the view should be declared in the `javascript` block and in this block, the file that will render the React components must be included in webpack. In `webpack.py` the following line must be added:
+
+```diff
+theme = WebpackThemeBundle(
+    __name__,
+    "assets",
+    default="semantic-ui",
+    themes={
+        "semantic-ui": dict(
+            entry={
++               "invenio-administration-search":
++                   "./js/invenio_app_rdm/src/search/search.js",
+                ...
+```
+
+Once the file was added to webpack we can add it to the template we are overriding. The webpack configuration **must** be registered in the module entry point!
+
+```html
+{% extends "invenio_administration/details.html" %}
+
+{% block admin_page_content %}
+  <div
+    id="invenio-details-config"
+  >
+  </div>
+{% endblock admin_page_content %}
+{% block javascript %}
+  {{ super() }}  # don't forget the parent JS assets!
+  {{ webpack['invenio-administration-search.js'] }}  # it is crucial to remember about adding proper assets to your jinja 
+{% endblock %}
+```
+
+When overriding the views, the block `admin_page_content` can be overridden, and a new `div` with a custom id can be set so that, in the new javascript file that was added to webpack the components can be injected in the view by looking for the correct id in the DOM and, like in the following example:
+
+```javascript
+const domContainer = document.getElementById("invenio-details-config");
+domContainer &&
+  ReactDOM.render(
+    <MyCustomComponent />,
+    domContainer
+  );
+
+```
+
+For most of the views there is no need to override the `admin_page_content` block as the React component can be injected in the already existing view - by reusing existing DOM and React root element IDs 
+Here a list of known DOM elements ids, depending on the template that's being extended:
+
+- `invenio_administration/create.html` -> id="**invenio-administration-create-root**"
+- `invenio_administration/details.html` -> id="**invenio-details-config**"
+- `invenio_administration/edit.html` -> id="**invenio-administration-edit-root**"
+- `invenio_administration/search.html` -> id="**invenio-search-config**"
+
+### Permissions
+
+The `administration-access` permission is needed to access the administration panel. This permission is being introduced in version 10. There are two ways to grant this permission.
```

Comment:
`Only users with action ``administration-access`` granted are allowed to access the Invenio Administration panel.`
This new permission's action has been introduced in InvenioRDM v10.0.0.

There are two ways to grant this permission: